README.md

```diff
@@ -13,29 +13,29 @@ This package is available in many languages in the standard packaging formats.
 To use from JavaScript or TypeScript in Node.js, install using either `npm`:
 
 ```
-$ npm install @jaxxstorm/pulumi-scaleway
+$ npm install @pulumiverse/pulumi-scaleway
```

Comment:
NPM packages typically drop the `pulumi-` part in the package name. Can you align and update this to `@pulumiverse/scaleway`?

Reply:
Fixed in latest commit

Reply:
@jaxxstorm I still see the wrong package name popping up.